test/test_utils.py

```diff
@@ -768,6 +768,15 @@ class MockCudaTimer(benchmark_utils.Timer):
             self.assertEqual(len(measurement.times), repeats)
             self.assertEqual(measurement.number_per_run, number_per_run)
 
+    @unittest.skipIf(not torch._C.valgrind_supported_platform(), "Unsupported platform.")
+    def test_collect_callgrind(self):
```

Comment:
How long does this test take? Might be worth marking it as a slow test

Reply:
About 30 seconds. That's a nice decorator. (which I added.)